chipy_org/settings.py

```diff
@@ -335,7 +342,8 @@ def env_list(key, defaults=None, delimiter=","):
 NH3_STRIP_COMMENTS = True  # default, listed here for documentation
 
 RECAPTCHA_PUBLIC_KEY = env_var("NORECAPTCHA_SITE_KEY")
-RECAPTCHA_PRIVATE_KEY = env_var("NORECAPTCHA_SECRET_KEY")
+RECAPTCHA_PRIVATE_KEY = env_var("NORECAPTCHA_PRIVATE_KEY")
```

Comment:
Do not change these.  If you want to use your own local envs values that's fine but they need to be set via the docker env vars, not with a code change.